src/rpc/rawtransaction.cpp

```diff
@@ -3076,9 +3090,17 @@ UniValue rawreissueasset(const JSONRPCRequest& request)
         throw JSONRPCError(RPC_INVALID_PARAMETER, "Transaction must have at least one output.");
     }
 
-    UniValue issuances = request.params[1].get_array();
+    // Validate fee output location, required by the implementation of reissueasset_base
+    if (!mtx.vout[mtx.vout.size() - 1].IsFee()) {
+        throw JSONRPCError(RPC_INVALID_PARAMETER, "Transaction must have exactly one fee output, which must be last");
```

Comment:
Might it make any sense to differentiate this exception text from the next (e.g., substituting or adding "Last output is not a fee output" vs. "fee output is not the last output")?  I don't have enough context/background to know for sure, but it seems like giving the user additional/discriminating information might be helpful for debugging.

Reply:
Hmm yeah I might agree with that. Exact same error messages for different pieces of code is often confusing. Even though I think Glenn carefully made sure the message should give clarity if you did either of the 2 wrong scenarios.

Reply:
Fair point, done.